contrib/images/Makefile

```diff
@@ -1,6 +1,7 @@
 RELAYER_TAG := $(shell grep '^ENV RELAYER_TAG' cosmos-relayer/Dockerfile | cut -f3 -d\ )
 BABYLON_FULL_PATH := $(shell git rev-parse --show-toplevel)
 BABYLON_VERSION_BEFORE_UPGRADE ?= v0.9.1
+E2E_INIT_CHAIN_VERSION_BEFORE_UPGRADE ?= v0.9.3
```

Comment:
do you think we can have only one variable like:
`BABYLON_VERSION_BEFORE_UPGRADE ` and set it to` v0.9.3` ?

Reply:
sure

Reply:
done at https://github.com/babylonlabs-io/babylon/pull/54/commits/52ca526bfa0fed136496a02fe42e34d35d71532c